generator/src/main/java/org/apache/kafka/message/FieldType.java

```diff
@@ -319,6 +322,10 @@ default boolean canBeNullable() {
         return Optional.empty();
     }
 
+    default boolean isVariableLength() {
+        return !fixedLength().isPresent();
```

Comment:
nit: maybe use `orElse(false)` to make it explicit (this works because the `fixedLength()` by default returns `Optional.empty()`, but I think making it explicit will be clearer to read)

Reply:
Hmm.  I don't think `orElse(false)` would work here because the value of `fixedLength` is `Integer`, and not `Boolean`.

Reply:
Ah, I missed that. Looks good as is then :-)